nodeup/pkg/model/kube_apiserver.go

```diff
@@ -157,6 +157,103 @@ func (b *KubeAPIServerBuilder) writeAuthenticationConfig(c *fi.ModelBuilderConte
 		return nil
 	}
 
+	if b.Cluster.Spec.Authentication.Heptio != nil {
```

Comment:
Perhaps left to another PR, but should consider merging the kopeio and this together given both is essentially just a web hook with a url diff ..

Reply:
Will create another PR to merge / cleanup.